### PR TITLE
create-entry first draft

### PR DESCRIPTION
**Pull request by Justin-Arakaki**
code-journal 1

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+function handleSubmitCreate(e) {
+  event.preventDefault();
+  const input = event.target.elements;
+  const newEntry = {
+    id: data.nextEntryId,
+    title: input.title.value,
+    photo: input.photo.value,
+    notes: input.notes.value
+  };
+  data.entries.push(newEntry);
```
Comment (BrettAlbright) [inline js/main.js]: this should be unshift, because you want the newest entry at the front of the array

File: js/main.js [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+  const dataJSON = JSON.stringify(data);
+  localStorage.setItem('data-local-storage', dataJSON);
```
Comment (BrettAlbright) [inline js/main.js]: local storage functionality should only be done when the page unloads, you don't want to directly access it here.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
A couple small updates!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Comment by Justin-Arakaki** [comment]
code-journal 2

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**